### PR TITLE
Fix multi

### PR DESCRIPTION
**Pull request by m-i-sadowski**
Changes required for multichannel

### CODE REVIEW

File: antha/anthalib/wtype/wellcoords.go [hunk 1]
```diff
@@ -115,7 +145,7 @@ func MakeWellCoords(wc string) WellCoords {
 
 // make well coordinates in the "A1" convention
 func MakeWellCoordsA1(a1 string) WellCoords {
-	if !MatchString("[A-Z]{1,}[0-9]{1,2}", a1) {
+	if !MatchString("^[A-Z]{1,}[0-9]{1,2}$", a1) {
```
Comment (ddn0) [inline antha/anthalib/wtype/wellcoords.go]: something like:
```go
re := regexp.MustCompile(`^([A-Z]+)([0-9]{1,2})$`)
matches := re.FindStringSubmatch(a1)
if matches == nil {
  return WellCoords{-1, -1}
}
X := wutil.ParseInt(matches[2]) - 1
Y := wunit.AlphaToNum(matches[1]) - 1
```

seems clearer to me

File: antha/compile/antha.go [hunk 2]
```diff
@@ -92,6 +92,7 @@ func (p *compiler) anthaInit() {
 		"MixNamed":      "execute.MixNamed",
 		"MixTo":         "execute.MixTo",
 		"ReadEM":        "execute.ReadEM",
+		"Wait":          "execute.Wait",
```
Comment (ddn0) [inline antha/compile/antha.go]: instructics.go:Wait is commented out but added here?

File: microArch/driver/liquidhandling/convertinstructions.go [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+
+package liquidhandling
+
+import (
+	"fmt"
+	"github.com/antha-lang/antha/antha/anthalib/wtype"
+	"github.com/antha-lang/antha/antha/anthalib/wunit"
+)
+
+//	this section aggregates instructions with the following constraints:
```
Comment (ddn0) [inline microArch/driver/liquidhandling/convertinstructions.go]: Comment unclear, perhaps flatten formatting. And move to `ConvertInstructions` which I believe this is documenting.

```
Without multichannel:
  input: LHIVectorA(A) ; LHIVectorB(B) ; LHIVectorC(C)
  output: XXX

With multichannel:
  input: LHIVectorD([[A A] [B] [C])
  output: TFR(A A d1 dest2), TFR(B dest2), TFR(C dest2), TFR(B dest1), TFR(C dest1) 

  Compared to without multichannel, we do LHIVectorA and first part of LHIVectorB in parallel ...

Not yet supported ...
```

File: target/mixer/opt.go [hunk 4]
```diff
@@ -45,6 +45,7 @@ type Opt struct {
 	DriverSpecificWashPreferences     []string
 	ModelEvaporation                  bool
 	OutputSort                        bool
+	UseNewExecutionPlanner            bool
```
Comment (ddn0) [inline target/mixer/opt.go]: difference from PlanningVersion?

File: microArch/driver/liquidhandling/transferinstruction.go [hunk 5]
```diff
@@ -0,0 +1,834 @@
...
+//
+//	The following is always mapped to single-channel operation
+//
+//		LHIVector([A,B], [A,C])
+//
+//	output:
+//		either case:
+//			SCB([A,B,A,C],[d1,d1,d2,d2])
+//
+
```
Comment (m-i-sadowski) [inline microArch/driver/liquidhandling/transferinstruction.go]: remove for godoc

**Pull request merged**